<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChronoDuino</class>
 <widget class="QMainWindow" name="ChronoDuino">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChronoDuino</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../common/icons/chrono.qrc">
    <normaloff>:/common/iconApp32.png</normaloff>:/common/iconApp32.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="m_action_new"/>
    <addaction name="m_action_open"/>
    <addaction name="m_action_preferences"/>
    <addaction name="separator"/>
    <addaction name="m_action_quit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="m_action_about"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="m_status_bar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>ToolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="m_action_test"/>
   <addaction name="m_action_print"/>
   <addaction name="m_action_quit"/>
  </widget>
  <action name="m_action_quit">
   <property name="icon">
    <iconset resource="../../../common/icons/chrono.qrc">
     <normaloff>:/common/exit.png</normaloff>:/common/exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="m_action_about">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="m_action_preferences">
   <property name="icon">
    <iconset resource="../../../common/icons/chrono.qrc">
     <normaloff>:/common/configure.png</normaloff>:/common/configure.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Preferences</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="m_action_open">
   <property name="text">
    <string>&amp;Open ...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="m_action_new">
   <property name="text">
    <string>&amp;New ...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="m_action_test">
   <property name="icon">
    <iconset resource="../../../common/icons/chrono.qrc">
     <normaloff>:/common/tester.png</normaloff>:/common/tester.png</iconset>
   </property>
   <property name="text">
    <string>Test serial</string>
   </property>
   <property name="toolTip">
    <string>Test serial</string>
   </property>
  </action>
  <action name="m_action_print">
   <property name="icon">
    <iconset resource="../../../common/icons/chrono.qrc">
     <normaloff>:/client/document-print-preview.png</normaloff>:/client/document-print-preview.png</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../../common/icons/chrono.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>m_action_quit</sender>
   <signal>triggered()</signal>
   <receiver>ChronoDuino</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
